docs/configure-and-manage/manage/qlight-node.md

```diff
@@ -57,11 +58,11 @@ This leverages the security model described under [JSON-RPC security](json-rpc-a
 When using [JSON-RPC security](json-rpc-api-security.md#enterprise-authorization-protocol-integration)
```

Comment:
In line 56, specify what "This" is. Also, don't use vague terms like "[leverage](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/l/leverage)."

E.g.:
```
The [enterprise authorization protocol integration](json-rpc-api-security.md#enterprise-authorization-protocol-integration) only allows authenticated clients to connect to the server.

Reply:
Fixed